Sources/PT.PM.Matching.Tests/CSharpMatchingTests.cs

```diff
@@ -59,5 +66,8 @@ public void Match_HardcodedPasswordAspx_WithoutException()
             match = matchResults.ElementAt(1).MatchedCode;
             Assert.IsTrue(match.Contains("PASSWORD") && match.Contains("hardcoded"));
         }
+
+        private FileCodeRepository CreateTestFileCodeRepo(string fileName)
```

Comment:
In other places, we don't shorten the repository word. Replace with `CreateTestFileCodeRepository`.